packages/nx/src/daemon/server/shutdown-utils.ts

```diff
@@ -63,6 +62,7 @@ export async function handleServerProcessTermination({
       );
     }
     serverLogger.log(`Server stopped because: "${reason}"`);
+    deleteDaemonJsonProcessCache();
```

Comment:
What was the significance of moving this? On second thought.. should this be here?

Reply:
Nothing, I was thinking this was affecting Windows